htmlFiles/input.html

```diff
@@ -0,0 +1,203 @@
+
+                <div class="input-validation-container">
+                        <label for="name" class="input-label fr-green">Name:</label>
+                    <input type="text" class="input-textbox error-validation" placeholder="Enter Your Name">
+                    <span class="error-message"> Please Enter your name</span>                                 
+                </div>
+
+                <div class="input-validation-container">
+                        <label for="name" class="input-label fr-green">Password:</label>
+                        <input type="text" class="input-textbox primary-validation" placeholder="Enter password">
+                        <span class="primary-message">password must be greater than 6 character</span>                                 
+                </div>
+
+                <div class="input-validation-container">
+                    <label for="name" class="input-label fr-green">Password:</label>
+                    <input type="text" class="input-textbox success-validation" placeholder="Enter password">
+                    <span class="success-message">This is success stage form validation</span>                                 
+                </div>
+</div>
+    <!------------------------------ code of input component---------------------------------------------- -->
+                <div class="section1-container">
+                                <h2 class="section-heading section-code-example-container">
+                                    <a href="">Code of Input Component</a>
+                                    <i class="fas fa-code" id ="input-code-btn"></i>
+                                </h2>
+
+                                <div class="code-snippet" id="input-div-code"> 
+                                    <iframe class="code-example"
+                                            src="https://carbon.now.sh/embed/cFmGymGBBdKAoqqHPyxY"
+                                            style="width: 700px; height: 300px; border:0; transform: scale(1); overflow:hidden;"
+                                            sandbox="allow-scripts allow-same-origin">
+                                     </iframe>
+
+                                     <iframe class="code-example"
+                                            src="https://carbon.now.sh/embed/qqwB3ycbF0ni4q6S1eki"
+                                            style="width: 700px; height: 300px; border:0; transform: scale(1); overflow:hidden;"
+                                            sandbox="allow-scripts allow-same-origin">
+                                    </iframe>
+                                </div>   
+                </div>
+  <!------------------------------------------------code of input ends- ---------------------------- -->
+         </section>
+    </div>
+<!-- ---------------------------------------------------------javascript functionality------------------------------- -->
+ <script>
```

Comment:
make another file of js and use it here via script tag

Reply:
i will try to add this